### PR TITLE
Feat:add contact page

### PR DESCRIPTION
**Pull request by fathahul-mihad**

### CODE REVIEW

**Comment by fathahul-mihad** [comment]
any comments

**Review by zeshhaan — CHANGES_REQUESTED** [review]
- align the form heading to the form content
image

- input text is very tiny, can you make it legible
image

File: contact.html [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+         </div>
+         <div class="form-group">
+           <label for="email">Email</label>
+           <input type="email" name="email" id="email" placeholder="Enter your email">
+         </div>
+         <div class="form-group">
+           <label for="message">How can we help you?</label>
+           <textarea name="message" id="message" placeholder="I was wondering about availability and rates. I need help with the following:"></textarea>
+         </div>
+         <input type="submit" value="Send" class="btn btn-dark">
```
Comment (zeshhaan) [inline contact.html]: - change it to button
- make the text similar to the one in the reference site. (capitalise it)
- add transition

File: contact.html [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+  <title>CONTACT | A K BROTHERS</title>
+</head>
+
+<body>
+  <!-- Hero section -->
+  <header class="contact-hero">
+    <div class="hero-content contact-container">
+      <div class="contact-heading hero-text-main">
+          Have a question for us?
+          Go for it.
```
Comment (zeshhaan) [inline contact.html:18]: add 'Go for it' on the next line. Check `<br>` tag

**Review by zeshhaan — COMMENTED** [review]
overall nice job, make the requested changes and send a PR. Thanks!

**Review by zeshhaan — CHANGES_REQUESTED** [review]
- give some space between the header image and main content
image

- also give space between each form groups. Currently they seems all stacked and jammed together and creates confusion among users on what label for what input
image


- align center
image

**Comment by zeshhaan** [comment]
@fathahul-mihad here is the embed code for map

> <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3895.3432945795043!2d74.98509531481706!3d12.49338999117091!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x3ba4826bc4743a25%3A0x1741f97f8ee5ed4e!2sA+K+Brothers+Marketing!5e0!3m2!1sen!2sin!4v1563350691573!5m2!1sen!2sin" width="100%" height="460" frameborder="0" style="border:0" allowfullscreen></iframe>

File: contact.html [hunk 3]
```diff
@@ -41,7 +42,7 @@ <h2 class="m-heading">Get in touch.</h2>
            <textarea name="message" id="message" placeholder="I was wondering about availability and rates. I need help with the following:"></textarea>
          </div>
          <div class="form-group">
-         <input type="submit" value="Send" class="btn-dark">
+         <input type="button" value="SUBMIT" class="btn-dark">
```
Comment (zeshhaan) [inline contact.html:45]: I know the referenced site is using an input field for form submit action. Can you instead make it a `button` element or `a` element?

**Review by zeshhaan — CHANGES_REQUESTED** [review]
#### Match it to the reference site adding padding to the x axes
image


####  Add space between `form-group` divs
@fathahul-mihad the space between each form groups is so thin, so i think it would be better off if we give space between each of the form-groups.
image

####  Also make the font label to `font-weight: 500;` instead of `font-weight: bold;`
image

**Comment by fathahul-mihad** [comment]
done some changes 
any comments?

**Comment by fathahul-mihad** [comment]
done

**Comment by zeshhaan** [comment]
> #### Match it to the reference site adding padding to the x axes
> image
> 
> #### Add space between `form-group` divs
> @fathahul-mihad the space between each form groups is so thin, so i think it would be better off if we give space between each of the form-groups. image
> 
> #### Also make the font label to `font-weight: 500;` instead of `font-weight: bold;`
> image

@fathahul-mihad Don't forget to make the font-size changes too

File: styles.css [hunk 4]
```diff
@@ -339,6 +372,103 @@ body {
...
+}
+
+#contact .contact-form .form-group {
+  margin: 0.75rem;
+}
+
+#contact .contact-form label {
+  display: block;
+  margin: 40px 0px 12px 0px;
+  font-weight: bold;
```
Comment (zeshhaan) [inline styles.css:401]: looks bit thicker, give it `500` instead of `bold`

**Comment by fathahul-mihad** [comment]
make some changes

**Pull request merged**